docs/DevGuide/Travis.md

```diff
@@ -74,6 +74,28 @@ performed in the CHECK_FILES build.
 using a Linux OS, and the `AppleClang` compiler for `OS X`.
 * The `gcc Debug` build will fail if there are `doxygen` warnings.
 
+## How to perform the checks locally
+
+Before pushing to GitHub and waiting for Travis to perform the checks it is
+useful to perform at least the following tests locally:
+- **Unit tests:** Perform a `make test-executables` and then `ctest -L unit` to
+  run all unit tests. As for `make` you can append a `-jN` flag to `ctest` to
+  run in parallel on `N` cores. To run only a subset of the tests you can use
+  one of the other keywords that the tests are labeled with, such as `ctest -L
+  datastructures`. To run only a particular test you can also execute `ctest -R
+  TEST_NAME` instead, where `TEST_NAME` is the test identifier such as
```

Comment:
for the -R option TEST_NAME can be a regular expression

You may want to mention that you can `ctest -h` to get a complete set of options